segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/tables/TableStore.java

```diff
@@ -41,8 +39,19 @@
  * will be atomically checked-and-applied.
  * * Unconditional Updates (insert, update, remove) will take effect regardless of what the current Key version exists in
  * the Table Segment.
+ *
+ * Sorted vs Non-Sorted Table Segments:
+ * * All Table Segments are a Hash-Table-like data structure, where Keys are mapped to Values.
+ * * Non-Sorted Table Segments provide no ordering guarantees for {@link #keyIterator} or {@link #entryIterator}.
+ * * Sorted Table Segments store additional information about the Keys and will return results for {@link #keyIterator}
+ * or {@link #entryIterator} in lexicographic bitwise order. All other contracts are identical to the Non-Sorted variant.
+ * * Sorted Table Segments will require additional storage space to store the ordered Keys and may require additional
```

Comment:
Do you want to add that this additional space is proportional to the number and length of keys, but not the size of the value?

Reply:
Good point. Done.